src/hazelcore/layout/DHDoc.re

```diff
@@ -7,11 +7,13 @@ type formattable_child = (~enforce_inline: bool) => t;
 
 let precedence_const = 0;
 let precedence_Ap = 1;
+let precedence_Subscript = 1;
 let precedence_Times = 2;
 let precedence_Divide = 2;
 let precedence_Plus = 3;
 let precedence_Minus = 3;
 let precedence_Cons = 4;
+let precedence_PlusPlus = 4;
```

Comment:
see above regarding precedence -- review all of these to make sure they are consistent with OCaml